testing/architecture-test/src/changes/archunit_store/public-api-properties-should-have-migration-annotation.txt

```diff
@@ -0,0 +1,128 @@
+Method <org.gradle.concurrent.ParallelismConfiguration.getMaxWorkerCount()> is not annotated with @NotToBeReplacedByLazyProperty in (ParallelismConfiguration.java:0) and Method <org.gradle.concurrent.ParallelismConfiguration.getMaxWorkerCount()> is not annotated with @ToBeReplacedByLazyProperty in (ParallelismConfiguration.java:0)
+Method <org.gradle.concurrent.ParallelismConfiguration.isParallelProjectExecutionEnabled()> is not annotated with @NotToBeReplacedByLazyProperty in (ParallelismConfiguration.java:0) and Method <org.gradle.concurrent.ParallelismConfiguration.isParallelProjectExecutionEnabled()> is not annotated with @ToBeReplacedByLazyProperty in (ParallelismConfiguration.java:0)
+Method <org.gradle.external.javadoc.JavadocOptionFileOption.getOption()> is not annotated with @NotToBeReplacedByLazyProperty in (JavadocOptionFileOption.java:0) and Method <org.gradle.external.javadoc.JavadocOptionFileOption.getOption()> is not annotated with @ToBeReplacedByLazyProperty in (JavadocOptionFileOption.java:0)
+Method <org.gradle.external.javadoc.OptionLessJavadocOptionFileOption.getValue()> is not annotated with @NotToBeReplacedByLazyProperty in (OptionLessJavadocOptionFileOption.java:0) and Method <org.gradle.external.javadoc.OptionLessJavadocOptionFileOption.getValue()> is not annotated with @ToBeReplacedByLazyProperty in (OptionLessJavadocOptionFileOption.java:0)
+Method <org.gradle.vcs.VersionControlRepository.getRootDir()> is not annotated with @NotToBeReplacedByLazyProperty in (VersionControlRepository.java:0) and Method <org.gradle.vcs.VersionControlRepository.getRootDir()> is not annotated with @ToBeReplacedByLazyProperty in (VersionControlRepository.java:0)
+Method <org.gradle.vcs.VersionControlSpec.getRepoName()> is not annotated with @NotToBeReplacedByLazyProperty in (VersionControlSpec.java:0) and Method <org.gradle.vcs.VersionControlSpec.getRepoName()> is not annotated with @ToBeReplacedByLazyProperty in (VersionControlSpec.java:0)
+Method <org.gradle.vcs.VersionControlSpec.getRootDir()> is not annotated with @NotToBeReplacedByLazyProperty in (VersionControlSpec.java:0) and Method <org.gradle.vcs.VersionControlSpec.getRootDir()> is not annotated with @ToBeReplacedByLazyProperty in (VersionControlSpec.java:0)
+Method <org.gradle.vcs.VersionControlSpec.getUniqueId()> is not annotated with @NotToBeReplacedByLazyProperty in (VersionControlSpec.java:0) and Method <org.gradle.vcs.VersionControlSpec.getUniqueId()> is not annotated with @ToBeReplacedByLazyProperty in (VersionControlSpec.java:0)
+Method <org.gradle.vcs.git.GitVersionControlSpec.getUrl()> is not annotated with @NotToBeReplacedByLazyProperty in (GitVersionControlSpec.java:0) and Method <org.gradle.vcs.git.GitVersionControlSpec.getUrl()> is not annotated with @ToBeReplacedByLazyProperty in (GitVersionControlSpec.java:0)
```

Comment:
What about the methods in this list? It seems there are only a few types here. Can we maybe mark the types themselves somehow? New annotation perhaps (`@NotToBeMigratedToLazy`)?